### PR TITLE
Multiple endpoints per namespace support

### PR DESCRIPTION
**Pull request by lulf**
### Type of change

- Bugfix
- Enhancement / new feature
- Refactoring
- Documentation

### Description

This modifies the infra to access a messaging project from multiple endpoints. This also allows the IoT to start using the internal listener using the namespace as the prefix for addresses @ctron 

Until there is a new router release, I have built a custom router image from qpid dispatch master source.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests ocp4

File: systemtests/pom.xml [hunk 1]
```diff
@@ -168,7 +168,7 @@
                     <groups>${groups}</groups>
                     <excludedGroups>${excludeGroups}</excludedGroups>
                     <skipTests>${skip.tests}</skipTests>
-                    <forkCount>1</forkCount>
+                    <forkCount>0</forkCount>
```
Comment (kornys) [inline systemtests/pom.xml:171]: :(

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot testcase=io.enmasse.systemtest.iot.isolated.http.HttpAdapterTest

File: systemtests/pom.xml (hunk 1 above)
Comment (lulf) [reply systemtests/pom.xml:171]: @kornys That was not intentional, just for local debugging. I will revert it :)

**Comment by lulf** [comment]
@enmasse-ci run tests ocp4

**Comment by lulf** [comment]
@enmasse-ci run tests ocp4

**Comment by lulf** [comment]
@enmasse-ci run tests ocp4

File: pkg/state/infra.go [hunk 2]
```diff
@@ -960,42 +1021,15 @@ func (i *infraClient) buildRouterEndpointEntities(endpoint *v1.MessagingEndpoint
 /**
  * Add router entities that should exist for a given address for a given endpoint.
```
Comment (k-wall) [inline pkg/state/infra.go:1022]: out of date comment

File: systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java [hunk 3]
```diff
@@ -205,7 +207,7 @@ public static boolean isOpenShiftCompatible() {
      * Check if tests are running on OpenShift (excluding CRC).
```
Comment (k-wall) [inline systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java:207]: I would expect the new test to pass on CRC

**Pull request merged**